examples/nlp/token_classification/conf/ner_config.yaml

```diff
@@ -0,0 +1,86 @@
+# Named Entity Recoginition with pretrained BERT models
+
+pl:
+  trainer:
+    gpus: 1 # the number of gpus, 0 for CPU
+    num_nodes: 1
+    max_epochs: 3
+    max_steps: null # precedence over max_epochs
+    accumulate_grad_batches: 1 # accumulates grads every k batches
+    amp_level: O0 # O1/O2 for mixed precision
+    distributed_backend: dpp
+
+model:
+  data_dir: ??? # /path/to/data
+  class_balancing: null # choose from [null, weighted_loss]. Weighted_loss enables the weighted class balancing of the loss, may be used for handling unbalanced classes
+
+  train_ds:
+    prefix: train
+    max_seq_length: 128
+    pad_label: 'O'
+    num_samples: -1
+    ignore_extra_tokens: false
+    ignore_start_end: false
+    overwrite_processed_files: false
+    shuffle: true
+    batch_size: 2
+    num_workers:  2
+    pin_memory: false
+    drop_last: false
+
+  validation_ds:
+    prefix: dev
+    max_seq_length: 128
+    pad_label: 'O'
+    num_samples: -1
+    ignore_extra_tokens: false
+    ignore_start_end: false
+    overwrite_processed_files: false
+    shuffle: false
+    batch_size: 2
+    num_workers:  2
+    pin_memory: false
+    drop_last: false
+
+  language_model:
+    pretrained_model_name: bert-base-uncased
+    bert_checkpoint: null
+    bert_config: null
+    tokenizer: nemobert # 'nemobert' or 'sentencepiece'
+    vocab_file: null
+    tokenizer_model: null # only used if tokenizer is sentencepiece
+    do_lower_case: false # true for uncased models, false for cased models, will be set automatically if pre-trained tokenizer model is used
```

Comment:
Shouldn't it be True as default bert is uncases?

Reply:
true is fine, with 'bert-base-uncased' and similar pretrained uncased models, tokenizer automatically does lower casing.